R/Rules-methods.R

```diff
@@ -3340,4 +3327,39 @@ setMethod("windowLength",
 
 ## ============================================================
 
+#nolint end
+
+##' --------------------------------------------------
+##' The maximum allowable relative increments in terms of DLTs
+##' --------------------------------------------------
+##' @describeIn maxDose Determine the maximum possible next dose based on
+##' relative increments determined by DLTs so far (modify increment rule in
+##' Rules-methods.R)
+setMethod("maxDose",
+  signature =
+    signature(
+      increments = "IncrementsRelativeDLTCumulative",
+      data = "Data"
+    ),
+  def =
+    function(increments, data, ...) {
+
+      # Determine what was the last dose.
+      lastDose <- tail(data@x, 1)
+
+      # Determine how many DLTs have occurred in last cohort.
+      lastCohort <- tail(data@cohort, 1)
+      index <- which(data@cohort == lastCohort)
+      dltHappened <- sum(data@y[index])
```

Comment:
@gunhanb Question: would we not want instead to sum the DLTs across all cohorts, i.e. cumulative?

Reply:
HI @danielinteractive you are abolsutely right. It is my mistake. The problem is about the documentation. What is implemented in crmPack is cumulative approach, but the extension is "not cumulative" actually.

Reply:
@danielinteractive Do you think now it can be merged?